js/data/bucket.js

```diff
@@ -282,7 +282,7 @@ Bucket.create = function(options) {
     let type = options.layer.type;
     if (type === 'fill' && (!options.layer.isPaintValueFeatureConstant('fill-extrude-height') ||
         !options.layer.isPaintValueZoomConstant('fill-extrude-height') ||
-        options.layer.getPaintValue('fill-extrude-height') !== 0)) {
+        options.layer.getPaintValue('fill-extrude-height', {zoom: this.zoom}) !== 0)) {
```

Comment:
This doesn't look right -- `this` here will be `Bucket` the class/constructor function, not a `Bucket` instance.